packages/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -173,6 +184,16 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
         return _byteCount * 16 + 21000;
     }
 
+    /// @notice Retuns the balance of the contract.
```

Comment:
TODO: comment about how this is guaranteed to be greater than or equal to the balance in L2, somebody can selfdestruct ether into the portal which will not trigger the deposit tx event